docs/core/compatibility/sdk/6.0/deprecate-p-option-dotnet-run.md

```diff
@@ -9,41 +9,25 @@ ms.date: 07/13/2021
 
 This warning comes from the CLI parser, so it won't generally cause failures when warnings are treated as errors. However, if your process wraps MSBuild or CI and checks for the text "Warning", the warning will appear in that check.
 
-Also, if you pass `--property` as an option to the application being run, the `--` syntax separator must be used.
-
 ## Version introduced
 
 .NET SDK 6.0.100 Preview 6
 
 ## Old behavior
 
-In previous .NET versions:
-
-- `-p` indicated `--project`.
-- `--property` was not a valid option on `dotnet run`. This option was passed through to the underlying application, even if you omitted the `--` syntax separator.
+In previous .NET versions, `-p` indicated `--project`.
 
 ## New behavior
 
-Starting in .NET 6:
-
-- Passing `-p` to `dotnet run` results in a warning that it is deprecated and to use the full `--project` option instead. Despite the warning, `-p` is still a valid abbreviation for `--project`, unless the argument includes an equals `=` character (in which case the option is interpreted as `--property`).
-- `--property` is now a valid option on `dotnet run`. This option is not passed through to the underlying application, unless you use the `--` syntax separator.
+Starting in .NET 6, passing `-p` to `dotnet run` results in a warning that it is deprecated and to use the full `--project` option instead. Despite the warning, `-p` is still a valid abbreviation for `--project`, unless the argument includes an equals `=` character (in which case the option is interpreted as `--property`).
 
 ## Reason for change
```

Comment:
I think it would be good to include more information about why were are making this change.  Maybe we can just link to the spec: https://github.com/dotnet/designs/pull/229